src/Formats/FormatSchemaFactory.h

```diff
@@ -22,7 +20,7 @@ class FormatSchemaFactory final : private boost::noncopyable
 
     void registerSchema(const String & schema_name, const String & format, std::string_view schema_body, ExistsOP exists_op, ContextPtr & context);
 
-    void unregisterSchema(const String & schema_name, const String & format, bool throw_if_not_exists, const ContextPtr & context);
+    void unregisterSchema(const String & schema_name, const String & format, bool throw_if_not_exists, ContextPtr & context);
```

Comment:
it is interesting we pass `ContextPtr &` around, shall it be `const ContextPtr & ` ?

Reply:
Fixed.